src/frontend/src/icp/api/bitcoin.api.ts

```diff
@@ -6,19 +6,21 @@ import { BitcoinCanister, type BitcoinNetwork, type get_utxos_response } from '@
 import { Principal } from '@dfinity/principal';
 import { assertNonNullish } from '@dfinity/utils';
 
+interface BitcoinCanisterParams {
```

Comment:
If you introduce an interface you can also use it in `const bitcoinCanister = async ({`

`Pick<BitcoinCanisterParams, "identity"...`

Reply:
Only for `bitcoinCanisterId`, because `identity` in `bitcoinCanister` function is `Identity` and not `OptionIdentity` (like the interface).

We assert the `nonNullish` identity in each extraction. Maybe I can do a separate PR where I assert it inside the function `bitcoinCanister`.

Reply:
Ah right, then it's not worth it. Nvm my comment. Thanks for the feedback.